sys/sys/_iovec.h

```diff
@@ -42,12 +42,10 @@ typedef	__size_t	size_t;
 #define	_SIZE_T_DECLARED
 #endif
 
-#ifndef _KERNEL
 struct iovec {
-	void *	iov_base;	/* Base address. */
+	void * __kerncap	iov_base;	/* Base address. */
 	size_t			iov_len;	/* Length. */
 };
-#endif
 #if __has_feature(capabilities)
 struct iovec_c {
```

Comment:
I wonder if we can hide iovec_c in a sys/compat/cheriabi header?  It might make it easier when it comes time to svn rm cheriabi if it is more self-contained.

Reply:
I agree, this would also mirror what is being done for iovec32 and iovec64.

Reply:
Done.